twisted/scripts/_twistd_unix.py

```diff
@@ -198,6 +199,38 @@ def preApplication(self):
         self.oldstderr = sys.stderr
 
 
+    def _formatException(self, exception):
+        """
+        Format the C{exception} in preparation for writing to the
+        status pipe.  This does the right thing on Python 2 if the
+        exception's message is Unicode, and in all cases limits the
+        length of the message, prior to encoding, to 100 codepoints.
+
+        This means the return message may be longer than 100 bytes.
```

Comment:
hm...we have a test... so this is a problem :) maybe the test can be re-written to check for size less than 150 to emphasize that the 100bytes is not a hard limit.